src/test/java/org/edumips64/CliRunnerTest.java

```diff
@@ -123,6 +123,14 @@ public void prints_usage_message_it() {
         assertFalse(os.toString().trim().isEmpty());
     }
 
+    @Test
+    public void prints_usage_message_zh() {
+        OutputStream os = getSystemOut();
+        Locale.setDefault(Locale.SCHINESE);
```

Comment:
```suggestion
        Locale.setDefault(Locale.SIMPLIFIED_CHINESE);
```

I think this should be `SIMPLIFIED_CHINESE`?
https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/util/Locale.html#SIMPLIFIED_CHINESE